packages/core/styles/_button-section.scss

```diff
@@ -3,7 +3,7 @@
   display: flex;
   justify-content: flex-end;
   margin: 0 1em; // align w/ data table
-
+  margin-left: auto;
```

Comment:
Consolidate with the top style and I believe we can remove the 1em. Test:

`margin: 0 0 0 auto`